src/OrbitPaths/Paths.cpp

```diff
@@ -199,17 +201,28 @@ ErrorMessageOr<std::filesystem::path> CreateOrGetOrbitUserDataDirSafe() {
   return path;
 }
 
-std::filesystem::path CreateOrGetLogDir() {
+static std::optional<std::filesystem::path> GetFlagLogDir() {
```

Comment:
"flag log" is a bit strange to me. What about `GetLogDirFromFlag`?

Reply:
sgtm. done!